ddtrace/ext/tags.go

```diff
@@ -106,10 +106,19 @@ const (
 
 	// SpanKind defines the kind of span based on Otel requirements (client, server, producer, consumer).
 	SpanKind = "span.kind"
+)
 
+// Messaging tags.
+const (
 	// MessagingSystem identifies which messaging system created this span (kafka, rabbitmq, amazonsqs, googlepubsub...)
 	MessagingSystem = "messaging.system"
 
+	// MessagingKafkaPartition defines the Kafka partition the trace is associated with.
+	MessagingKafkaPartition = "messaging.kafka.partition"
+)
+
+// RPC tags
+const (
```

Comment:
I think this could be moved to a separate file in `rpc.go` to keep it organized.

Reply:
I think we agreed to do that at the very end with a tag refactoring, but for now lets just keep it all in one place